package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "coderscamp2020.project.typescript.pokemonfight",
+  "version": "1.0.0",
+  "description": "**UWAGA! Zaczynając pracę nad projektem — nie róbcie forka.\r Jedna osoba z zespołu (np. Mentor) powinna użyć przycisku `Use this template` i dodać innych członków zespołu jako Collaborators do tego repozytorium.\r Mentorzy mogą dowolnie zmieniać zakres projektu lub zdecydować się na zupełnie inny temat.**",
+  "main": "index.ts",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "build": "webpack",
+    "start": "webpack --watch"
```

Comment:
so we decided to use webpack, not parcel? Is there a difference? :P

Reply:
It looks like Parcel is more beginner-friendly, with it's autoconfigs:
https://stackoverflow.com/questions/51453462/webpack-vs-parcel

Reply:
It's just that we had webpack in our basic study materials, so it should be already familiar to us